Fix: Managed access flag is missing from AnVIL deployments (#6754) (#6827)
DataBiosphere/azul: UPGRADING.rst

reverted. This is all fairly informal and loosely defined. Hopefully we won't
have too many entries in this file.

#6754 Managed access flag is missing from AnVIL deployments
===========================================================

> We're getting rid of the ``ma`` flag in all deployments. Remove the ``ma`` flag
> and related logic off your personal deployment using the sandbox deployment's
> ``environment.py`` as a model.

Reviewer: ```suggestion
Using the sandbox deployment's ``environment.py`` as a model, remove the ``ma``
flag and related logic from your personal deployment's ``environment.py``.
```
